tests/baselines/reference/targetTypeTest1.errors.txt

```diff
@@ -14,7 +16,11 @@ tests/cases/compiler/targetTypeTest1.ts(60,10): error TS2300: Duplicate identifi
           public x: number;
           public y: number;
           public add(dx: number, dy: number): Point;
+                                              ~~~~~
+!!! error TS2709: Cannot use namespace 'Point' as a type.

```

Comment:
shouldn't the containing `declare class Point` give a type meaning to this symbol? I thought that the first declaration won, but maybe it's the non-ambient one?

Reply:
`function Point` was the chosen declaration before (functions are bound early, classes and functions cannot merge) - this only "worked" because the `Assignment` flag (added by the `Point.origin` assignment) conferred a `Type` meaning before - the type on inspection could only have been the `errorType`. Now `Assignment` no longer adds any extraneous meaning, so this is correctly marked as an error.